cli/src/describe.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::{Error, Result};
+use clap::Parser;
+use model::test_manager::TestManager;
+
+/// Retrieve the YAML description of a test or resource.
+#[derive(Debug, Parser)]
+pub(crate) struct Describe {
+    /// The name of the test we want the description from.
+    #[clap(long, conflicts_with = "resource")]
+    test: Option<String>,
+
+    /// The name of the resource we want the description from.
+    #[clap(long, conflicts_with = "test")]
+    resource: Option<String>,
+}
+
+impl Describe {
+    pub(crate) async fn run(self, client: TestManager) -> Result<()> {
+        match (self.test, self.resource) {
+            (Some(test), None) => {
+                let description = client.describe_test(test).await?;
```

Comment:
We can get the test with 
`client.test_client().get(<test name>)`
and convert to string here.

Reply:
Another option would be to add a `to_string` or `to_yaml` method to `crd_ext.rs` and call `client.list(&SelectionParams::Name(CrdName::Test(<test name>))).first().to_string()` from `describe.rs`. It doesn't look as clean, but I'm not sure if the CLI should know about/interact with `TestClient` (seeing as every other command only interacts with `TestManager` as well). If this is not a concern, however, I'd be happy to change it.

Reply:
In the case of getting a single item. We should use `get` instead of `list`. When `TestManager` was created it was decided that methods from existing clients would not be duplicated i.e `test_manager::get_test()` because we already have `test_client::get()` and that a user would use the manager to get the client and then use the client to get the object (the logic behind that decision may need to be reconsidered). I think adding a function to `crd_ext` to get the yaml form of the object is a good start.

Reply:
I agree that using `list` is not great for a single item. I do think that there should be a `get_test()` method in `test_manager`, but that is up for discussion. For now, I can add the function to `crd_ext` and have the `manager` get and print the CRD via the `test_client`.